c_api/cmake/FindTaichi.cmake

```diff
@@ -81,7 +81,7 @@ else()
 endif()
 
 # Set up default find components
-if("${taichi_FIND_COMPONENTS}" STREQUAL "")
+if(TRUE)
```

Comment:
ATM we only have taichi runtime so we don't check the components. This could be useful when we ship various things besides the runtime including the codegen in the future.